CMakeLists.txt

```diff
@@ -95,6 +95,11 @@ include("${PCL_SOURCE_DIR}/cmake/pcl_verbosity.cmake")
 include("${PCL_SOURCE_DIR}/cmake/pcl_targets.cmake")
 include("${PCL_SOURCE_DIR}/cmake/pcl_options.cmake")
 include("${PCL_SOURCE_DIR}/cmake/clang-format.cmake")
+include("${PCL_SOURCE_DIR}/cmake/pcl_utils.cmake")
+DISSECT_VERSION()
+GET_OS_INFO()
+SET_INSTALL_DIRS()
```

Comment:
I would appreciate adopting lower-case commands in new/moved CMake code.

Reply:
I don't mind, do you have any pointers that this is "good practice or examples where this is done" 👍

Reply:
> Just glanced at methods at VTK - and indeed they are seemingly all lowercase - I'll keep it in mind.

VTK... It is maintained by Kitware as is CMake. It is probably an old CMake code base, and it tells me that Kitware should ... have a lot of early awareness of shortcomings in CMake.
(Coincidentally I'm trying to fix issues in the vtk port in vcpkg.)